src/awx_plugins/credentials/aim.py

```diff
@@ -111,6 +111,19 @@ def aim_backend(**kwargs):
             verify=verify,
             allow_redirects=False,
         )
+    sensitive_query_params = {
+        'AppId': '****',
+        'Query': '****',
+        'QueryFormat': object_query_format,
+    }
+    if reason:
+        sensitive_query_params['reason'] = '****'
+    sensitive_request_qs = (
+        '?' +
+        urlencode(sensitive_query_params, safe='*', quote_via=quote)
+    )
+    res.url = request_url + sensitive_request_qs
```

Comment:
I'd recommend moving the delimiter to the concatenation instruction. It's also not a part of a query string per RFCs so putting it into a variable called like this would be counterintuitive/misleading:
```suggestion
    sensitive_request_qs = urlencode(
        sensitive_query_params,
        safe='*',
        quote_via=quote,
    )
    res.url = f'{request_url}?{sensitive_request_qs}'
```
P.S. f-strings are always more performant than any other concatenation methods, use them whenever possible.